autogluon/scheduler/hyperband.py

```diff
@@ -219,6 +218,9 @@ def _run_reporter(self, task, task_job, reporter, searcher, terminator):
         last_updated = None
         while not task_job.done():
             reported_result = reporter.fetch()
+            if reported_result is None:
```

Comment:
Same issues as above. I do think reporter.fetch() would not return None. The question is whether it would return an empty dict.